src/commands.rs

```diff
@@ -2435,20 +2437,54 @@ fn cmd_resolve(
 
 fn print_conflicted_files(
     conflicts: &[(RepoPath, jujutsu_lib::backend::ConflictId)],
+    tree: &Tree,
     formatter: &mut dyn Formatter,
     workspace_command: &WorkspaceCommandHelper,
 ) -> Result<(), CommandError> {
-    for (repo_path, _conflict_id) in conflicts.iter() {
-        // TODO: Similar to `jj diff --summary`, insert a few letters
-        // before the filename to indicate the kind of conflict.
-        // E.g. we could have a letter per add : `FF` is a usual conflict
-        // between two versions of a file, `FD` is a file vs directory,
-        // `FFF` for a merge of three conflicting versions. Additionally,
-        // if (# removes) + 1 > (# adds), this indicates the file was deleted
-        // in some versions of the conflict. Perhaps that should be `R` for removed.
+    for (repo_path, conflict_id) in conflicts.iter() {
+        let conflict = tree.store().read_conflict(repo_path, conflict_id)?;
+        let sides = conflict.adds.len();
+        let deletions: i32 =
+            1 + i32::try_from(conflict.removes.len()).unwrap() - i32::try_from(sides).unwrap();
```

Comment:
Is there a better way to do arithmetic?

Reply:
Actually, I was over-complicating things. No need for arithmetic that complicated.